app/src/main/java/chat/rocket/android/util/SerialisationUtils.kt

```diff
@@ -0,0 +1,22 @@
+package chat.rocket.android.util
+
+import timber.log.Timber
+import java.io.*
+
+var byteArray: ByteArray? = null
+
+fun serialiseToken(token: TokenSerialisableModel): ByteArray? {
+    val a = ByteArrayOutputStream()
```

Comment:
Avoid creating meaningless variables, like you did here for the variables `a` and `b`.